index.js

```diff
@@ -83,51 +81,32 @@ const nonWindowsSingleCall = async (options = {}) => {
 	});
 
 	const lines = stdout.trim().split('\n');
+	const COMMAND_POSITION = lines[0].lastIndexOf('COMMAND') - lines[0].indexOf('COMMAND');
 	lines.shift();
 
-	let psIndex;
-	let commPosition;
-	let argsPosition;
-
-	const processes = lines.map((line, index) => {
+	return lines.map(line => {
 		const match = psOutputRegex.exec(line);
 		if (match === null) {
 			throw new Error(ERROR_MESSAGE_PARSING_FAILED);
 		}
 
-		const {pid, ppid, uid, cpu, memory} = match.groups;
+		const {pid, ppid, uid, cpu, memory, commAndArgs} = match.groups;
+		const comm = commAndArgs.slice(0, COMMAND_POSITION).trim();
+		if (psPID === pid) {
+			return null;
```

Comment:
```suggestion
			return;
```

I don't use `null`: https://github.com/sindresorhus/meta/issues/7